scripts/scil_convert_sh_basis.py

```diff
@@ -42,13 +49,18 @@ def main():
     assert_inputs_exist(parser, args.in_sh)
     assert_outputs_exist(parser, args, args.out_sh)
 
+    print(args.is_input_not_legacy)
```

Comment:
Remove print and add logging to verbose some info of the sh basis/legacy/not legacy...

Reply:
Not sure the verbose is necessary here... and if so, what would be the right way to do it?